server/data/orchestrationApiTypes.ts

```diff
@@ -27,3 +27,9 @@ export type SessionCapacity = components['schemas']['SessionCapacityDto']
 export type SessionSchedule = components['schemas']['SessionScheduleDto']
 
 export type VisitSession = components['schemas']['VisitSessionDto']
+
+export type Alert = components['schemas']['AlertDto']
+
+export type PrisonerProfile = Omit<components['schemas']['PrisonerProfileDto'], 'visits'> & {
```

Comment:
If we pull in changes from main (I guess via `develop`) then we no longer need this workaround and the ones above will go to.